msgcntr/messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/AnonymousMapping.hbm.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping>
+	<class name="org.sakaiproject.component.app.messageforums.dao.hibernate.AnonymousMappingImpl" table="MFR_ANONYMOUS_MAPPING_T">
+		<id column="ID" name="id" type="long">
```

Comment:
Check whether ID is required, if not, I suggest removing it and relying on SITE_ID + USER_ID as the unique ID. Also, make sure that there is an index created for the SITE_ID + USER_ID combination